### PR TITLE
Add dynamic build date using time() function in VersionInfo

### PR DESCRIPTION
**Pull request by AzaleeX**
This update introduces a dynamic build date in the `VersionInfo` class using the `time()` function, allowing the build date to be set at runtime. This change ensures that the build date is always current without needing manual updates.

- remove from Todos list

### CODE REVIEW

**Comment by jasonw4331** [comment]
The server isn't built during runtime. It's built during build time. Please ensure the build time is accurate to a time generated during the build step rather than when the server is ran.

File: src/VersionInfo.php [hunk 1]
```diff
@@ -103,4 +104,8 @@ public static function VERSION() : VersionString{
 		}
 		return self::$fullVersion;
 	}
+
+	public static function BUILD_DATE() : int {
+		return time();
```
Comment (dktapps) [inline src/VersionInfo.php]: It's supposed to be the time the phar was built, not the time the function was called.
This isn't as simple as just adding an extra function.

**Review by dktapps — CHANGES_REQUESTED** [review]

File: src/VersionInfo.php [hunk 2]
```diff
@@ -103,4 +104,21 @@ public static function VERSION() : VersionString{
...
+	//I don't know if it would really work
+	public static function BUILD_DATE() : int {
+		static $buildDate = null;
+		if ($buildDate === null) {
+			$buildDate = time();
+			if (\Phar::running(true) !== "") {
+				$pharPath = \Phar::running(false);
+				$phar = \Phar::isValidPharFilename($pharPath) ? new \Phar($pharPath) : new \PharData($pharPath);
+				$meta = $phar->getMetadata();
+				if (isset($meta["date"])) {
```
Comment (dktapps) [inline src/VersionInfo.php:117]: This isn't going to magically appear in the phar metadata

**Comment by dktapps** [comment]
Closing due to lack of activity on review comments and little need for this change anyway

**Pull request closed without merging**